### PR TITLE
docs: command line reference

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates to keymanapp/keyman.com#306

### CODE REVIEW

File: developer/14.0/guides/command-line.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+`source/readme.htm`
+: Introductory web page for end users of your keyboard, shown before they
+  install it, so they know what it is used for.
+
+`source/welcome.htm`
+: A web page which describes how to use your keyboard, ideally with examples.
+
+[`source/sample.keyman-touch-layout`](../reference/file-types/keyman-touch-layout)
+: This touch layout file description is in JSON format and most easily visually
+  edited with the Keyman Developer touch layout editor. This file is optional;
```
Comment (darcywong00) [inline developer/14.0/guides/command-line.md:175]: Isn't this file required for touch layout keyboards?

File: developer/14.0/guides/command-line.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+already be compiled.
+
+* [KMComp reference](../context/kmcomp)
+
+### KMConvert
+
+[KMConvert](../context/kmconvert) generates keyboards and models from templates,
+and converts keyboard layouts between different formats.
+
+The [New Project dialog](new-project) in Keyman Developer provides a graphical
```
Comment (darcywong00) [inline developer/14.0/guides/command-line.md:231]:
```suggestion
The [New Project dialog](../context/new-project) in Keyman Developer provides a graphical
Write
```

File: developer/14.0/guides/command-line.md (hunk 1 above)
Comment (mcdurdin) [reply developer/14.0/guides/command-line.md:175]: For sure, but if you create a desktop layout it's not required...

File: developer/14.0/guides/command-line.md (hunk 2 above)
Comment (mcdurdin) [reply developer/14.0/guides/command-line.md:231]: Did you mean to add the word `Write` there?

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**